packages/lib-react-components/src/PlainButton/PlainButton.js

```diff
@@ -52,36 +54,29 @@ function PlainButton (props) {
   )
 }
 
-PlainButton.defaultProps = {
-  as: '',
-  className: '',
-  color: {
-    dark: 'accent-1',
-    light: 'neutral-1'
-  },
-  href: '',
-  labelSize: 'medium',
-  onClick: () => {},
-  text: '',
-  theme: {
-    dark: false
-  }
-}
-
 PlainButton.propTypes = {
+  /** (string) The DOM tag or react component to use for the Grommet button element. */
   as: string,
   className: string,
+  /** ({ dark: string, light: string}): Defaults to { dark: 'accent-1', light: 'neutral-1' } */
```

Comment:
I think `docgen` will pick up the default value from the destructured function arguments, so you don't need to edit it in two places here.

Reply:
Yeah. The storybook already shows the defaults for each prop, so duplicating that in the description seems redundant.

Right now, the 'Default' column is showing nothing for each prop, so something's wrong in the component code somewhere.

<img width="1019" alt="Documentation for the PlainButton props in the storybook." src="https://user-images.githubusercontent.com/59547/167617442-134ead92-0ea8-4190-b3b9-3011b91ffdef.png">

Reply:
My fault. I was on an outdated copy of this branch. Here you can see that the type and default value are duplicated in the description.

<img width="1052" alt="Updated screenshot showing prop types and defaults for the PlainButton props in the storybook." src="https://user-images.githubusercontent.com/59547/167617951-3cfbf844-5cb5-40b7-9c10-a8b6f870db16.png">